sample/src/main/webapp/WEB-INF/jboss-ejb3.xml

```diff
@@ -25,6 +25,10 @@
 
   <assembly-descriptor>
     <s:security>
+      <ejb-name>AccountsMessageListener</ejb-name>
```

Comment:
This is a small hack... Unfortunately, our MDB doesn't get any security context, and the container blocks the execution of it, as the KC security-domain has the auth as "required". So, we switch this specific bean to the "other" security domain, which can be executed as anonymous and with RunAs